tests/examples/hello/src/square.cc

```diff
@@ -0,0 +1,33 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class SquareTask : public Task<SquareTask, SQUARE> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &output = context.outputs()[0];
+    auto output_shape = output.shape<1>();
+    auto out = output.write_accessor<float, 1>();
+
+    auto &input = context.inputs()[0];
+    auto input_shape = input.shape<1>(); // should be a 1-Dim array
+    auto in = input.read_accessor<float, 1>();
+
```

Comment:
I suggest adding an assert here, which verifies that `add_alignment` worked as expected; i.e. every point task will received aligned spans over the two arrays.
```
assert(input_shape == output_shape);
```